packages/cli/src/lib/implementation/core-config.middleware.ts

```diff
@@ -1,23 +1,24 @@
 import { autoloadRc, readRcByPath } from '@code-pushup/core';
 import {
-  CoreConfig,
   PERSIST_FILENAME,
   PERSIST_FORMAT,
   PERSIST_OUTPUT_DIR,
+  uploadConfigSchema,
 } from '@code-pushup/models';
+import { CoreConfigCliOptions } from './core-config.model';
 import { GeneralCliOptions } from './global.model';
 
 export async function coreConfigMiddleware<
-  T extends Partial<GeneralCliOptions & CoreConfig>,
+  T extends Partial<GeneralCliOptions & CoreConfigCliOptions>,
```

Comment:
Could we name it RcConfig? I is covering the type of the rc file.

Reply:
The way I understand it, this is the type for CLI arguments only that are passed to the function, not the config file which is loaded inside of this function and returned (hence why I added the return type as `CoreConfig` in my latest commit).

So I am not sure what you mean by this, can you explain?